plugin/src/compilation/decompiler.ml

```diff
@@ -0,0 +1,89 @@
+
+open Names
+open Constr
+open Environ
+open Envutils
+open Tactics
+open Pp
+open Contextutils
+
+(* Abstraction of Coq tactics supported by this decompiler. *)
+type tact =
+  | Intro of Id.t
+  | Intros of Id.t list
+  | Apply of env * types
+        
+(* Return the string representation of a single tactic.
+   TODO: Indentation/bulleting
```

Comment:
Can you address these before pushing?

Reply:
What should I address in this code snippet? Is it the comments?

Reply:
Yeah, just either address the TODOs or make Github issues for them. In general, I try to avoid letting TODOs leak into master, though there are many still there from the early days of development before I enforced this on myself :)

Reply:
Good idea! I'll start doing that.